tests/conftest.py

```diff
@@ -35,7 +35,9 @@ def app():
 
 @pytest.fixture
 def testapp(app):
-    return TestApp(app)
+    _app = TestApp(app)
+    _app.authorization = ('Basic', ('username', 'password'))
```

Comment:
Ah, cool. I think that addresses my concern and also keeps the tests from turning into a mess of boilerplate. :+1: